kong/acl_service_test.go

```diff
@@ -137,6 +137,58 @@ func TestACLGroupGet(T *testing.T) {
 	require.NoError(client.Consumers.Delete(defaultCtx, consumer.ID))
 }
 
+func TestACLGroupGetByID(T *testing.T) {
```

Comment:
There's no reason to name a local variable with a capital letter - let's just use `t` as that's the common pattern.
```suggestion
func TestACLGroupGetByID(t *testing.T) {
```

Reply:
Done, the existing tests had the variable with capital letter, so I had followed the same initially.